api/push.py

```diff
@@ -127,7 +133,9 @@ def post(self):
                 self.get_apns_conn().process(token=self.token, alert=alert, extra=extra, apns=data['apns'])
                 self.send_response(ACCEPTED)
             elif device == DEVICE_TYPE_ANDROID:
-                data.setdefault('gcm', {})
+                # jwk- 2017-10-31
+                if ('gcm' not in data):
+                    data.setdefault('gcm', {})
```

Comment:
thanks for fix this issue @jkalkhof we could keep this line 136-138 only for this PR, other logging code are not necessary imho, your name and date stays in commit history, so the comment is not necessary as well :)